acceptance/acceptance_test.go

```diff
@@ -65,6 +66,34 @@ func TestInprocessMode(t *testing.T) {
 	require.Equal(t, 1, testAccept(t, true, "selftest"))
 }
 
+// Cache terraform binary in the temporary directory to avoid downloading it in each test.
```

Comment:
Thanks, this is indeed a needed feature, both for local tests but most importantly for cloud tests (https://github.com/databricks/cli/pull/2242).

Could you make a separate PR just for cached terraform - it's important that we review it well.

Also, could you not depend on side effect of "bundle validate"? Here some scripts/docs on how to prepare terraform.

- https://gist.github.com/pietern/1cb6b6f3e0a452328e13cdc75031105e
- https://databricks.atlassian.net/wiki/spaces/UN/pages/3317564067/Setting+up+DABs+in+an+airgapped+environment+go+airgapped-dabs

Reply:
I've seen instructions for air-gapped environments. I'm not sure I will have enough capacity to do it correctly. I think we need to implement it by relying on caching in GitHub actions and adding a step where we download Terraform. The cache key should be based on Terraform and Terraform provider versions. Right now, we don't even do that in our own GitHub action that we provide.

Reply:
I think Github actions cache is nice to have, not a must.

> I'm not sure I will have enough capacity to do it correctly.

Let's discuss on Slack -- I can help with that.

Reply:
Please rebase on top of this https://github.com/databricks/cli/pull/2267